Purchases/Public/PurchasesDelegate.swift

```diff
@@ -24,14 +24,22 @@ import StoreKit
  */
 @objc(RCPurchasesDelegate) public protocol PurchasesDelegate: NSObjectProtocol {
 
+    /**
+     * - Note: Deprecated, use purchases(_ purchases: Purchases, receivedUpdated customerInfo: CustomerInfo) or
+     * objc: purchases:receivedUpdatedCustomerInfo:
+     */
+    @available(swift, deprecated: 5, obsoleted: 5, renamed: "purchases(_:receivedUpdated:)")
```

Comment:
what does this do for objective-c?

Reply:
if it doesn't work for objc I think we can do something like `@available(iOS, macOS, tvOS, watchOS, deprecated:...`

Reply:
Yeah, you're on the right track, unfortunately: https://twitter.com/taquitos/status/1443742729612058624
`@available(iOS, obsoleted: 1, renamed: "taquitos()")`
I have to do each platform separately because `*` can't be used.